### PR TITLE
React Calcultator Milestone1 Code Review

### PR DESCRIPTION
**Pull request by majovanilla**
This pull request includes:

- Initial application setup
- Installation of `prop-types` and `big.js` using npm
- Deployment to Heroku [here](https://react-calculator-heroku.herokuapp.com/)
- Linters setup

### CODE REVIEW

File: eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "env": {
```
Comment (bellom) [inline eslintrc.json]: You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)
You need to fix linter's error without changing the file
_Hint: you can make use of in-line comment to ignore some tricking issue._
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: package.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```
Comment (bellom) [inline package.json]: **In this project, you will be making use of `big-js` not `big.js`**
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/
**Kindly update this...**

File: eslintrc.json (hunk 1 above)
Comment (majovanilla) [reply eslintrc.json]: Thanks for the review, but I didn't make any changes. If you check the file, you will see it's the same.

File: package.json (hunk 2 above)
Comment (majovanilla) [reply package.json]: I installed the package using the link on the same page that you mentioned and followed the instructions in the official page. So the package should be correct.

**Comment by majovanilla** [comment]
There is no linter error. Stickler is working just fine and if you see the checks, it shows everything passed. The yellow color is related to another check unrelated to the project.

**Pull request merged**